apps/crn-server/src/controllers/research-outputs.ts

```diff
@@ -152,10 +153,35 @@ export default class ResearchOutputs implements ResearchOutputController {
   }
   private async createResearchOutput({
     subTypes,
+    identifierType,
+    identifier: rawIdentifier,
     ...researchOutputData
   }: Omit<ResearchOutputPostRequest, 'teams'>) {
+    let identifier:
```

Comment:
maybe extract this to a function? this will make the code a little cleaner (ie no switch, let, etc)